examples/customresources/models/mysum/mysum.go

```diff
@@ -59,8 +64,13 @@ func (m *mySum) Sum(ctx context.Context, nums []float64) (float64, error) {
 }
 
 func (m *mySum) Reconfigure(ctx context.Context, deps resource.Dependencies, conf resource.Config) error {
+	sumConfig, err := resource.NativeConfig[*Config](conf)
+	if err != nil {
+		return err
+	}
+
 	m.mu.Lock()
 	defer m.mu.Unlock()
-	m.subtract = conf.Attributes.Bool("subtract", false)
+	m.subtract = sumConfig.Subtract
```

Comment:
make false by default?

Reply:
omitempty in config, that defaults to false

Reply:
Ah, that works. But should comment that bit of the config structs. JSON tags are also non-obvious.